### PR TITLE
Bugs fixed

### LINKED ISSUE #134: [Bug] Fall damage view command inconsistant
# Bug Report

## Version info

Zone Version: 1.0.0.0
Sponge API: 8.0.0
Sponge impl: Sponge Vanilla
Minecraft version: MC 1.16.5

## Description of bug

>> In the current version of zones, all fall damage flag commands are ``zone region flag <zone> damage fall`` however the view command is ``zone region flag <zone> fall damage

## How to reproduce?

>> Attempt to run the command

### LINKED ISSUE #135: [Bug] View damage flag command inconsistant
# Bug Report

## Version info

Zone Version: 1.0.0.0
Sponge API: 8.0.0
Sponge impl: Sponge Vanilla
Minecraft version: MC 1.16.5

## Description of bug

>> the view attack flag is currently ``zone region flag <zone> damage player view`` however it should be ``zone region flag <zone> damage attack player view``

## How to reproduce?

>> Attempt to run the command

### LINKED ISSUE #137: [Bug] Attack damage group cannot be changed
# Bug Report

## Version info

Zone Version: 1.0.0.0
Sponge API: 8.0.0
Sponge impl: Sponge Vanilla
Minecraft version: MC 1.16.5

## Description of bug

>> Missing the command to change the group

## How to reproduce?

>> Attempt to run the command

### PR DESCRIPTION
**Pull request by Lukamon24**
Check it ; )

### CODE REVIEW

**Comment by Lukamon24** [comment]
ahha

File: src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java [hunk 1]
```diff
@@ -49,7 +49,8 @@
         commandContext
                 .getCause()
                 .sendMessage(Identity.nil(),
-                             Component.text("Enabled: " + zone.containsFlag(FlagTypes.PREVENT_PLAYERS)));
+                             Component.text("Enabled: " + zone.containsFlag(FlagTypes.PREVENT_PLAYERS))
+                                     .color(NamedTextColor.AQUA));
```
Comment (mosemister) [inline src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java:53]: We now have a messages for this message

``commandContext.sendMessage(Messages.getEnabledInfo(zone.containsFlag(FlagTypes.PREVENT_PLAYERS)));``

Comment (Lukamon24) [reply src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java:53]: ohoh

Comment (Lukamon24) [reply src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java:53]: group?

Comment (mosemister) [reply src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagPlayerEntryViewCommand.java:53]: There is one for group too, take a look at the other view commands that have the updated message, but im not too bothered about the group one as we are going to do another sweep of messages

**Review by mosemister — APPROVED** [review]
While messages can be taken from Messages.java, im not too bothered as we are doing a sweep later, so ill approve

**Comment by Lukamon24** [comment]
k

**Pull request merged**